presenters/slice_plotter_presenter.py

```diff
@@ -27,6 +42,8 @@ def __init__(self, slice_view, slice_plotter):
         self._slice_view = slice_view
         self._main_presenter = None
         self._slice_plotter = slice_plotter
+        colormaps = self._slice_plotter.get_available_colormaps()
```

Comment:
Good style here using the modelto provide the data that is pushed in to the view.